### DISCUSSION TITLE
imgui_internal.h missing implementations

### DISCUSSION THREAD

**Discussion by AdlanSADOU** [ocornut/imgui#6198] [New Users Build/Link/Run issues ONLY!]
Hi, I'm trying to integrate imgui into my Vulkan renderer with SDL2 in Visual Studio 2022. 

My problem is that there are missing implementations defined in imgui_internal.h
I have cloned and searched the repo for the missing functions, and did not find them.
just to cite a few:

```
ImGuiMenuColumns::Update
ImGui::EndColumns
ImGui::TableSetBgColor
```

I searched through issues and discussions without luck.
I went through the comments in imgui.cpp and imgui_internal.h without luck.
Yes, I have included the sources files into my project.

Am I missing some unspoken assumptions?

The source and header files I have included are:

```
imgui.h
imgui.cpp
imgui_internal.h

imgui_draw.cpp

imgui_impl_sdl2.h
imgui_impl_sdl2.cpp

imgui_impl_vulkan.h
imgui_impl_vulkan.cpp
```

**Comment by ocornut** [comment]
> I searched through issues and discussions without luck.

when a function is not linked search for its name in the codebase. You’ll easily find that those are all implemented in imgui_tables.cpp which you have omitted from your build, along with imgui_demo.cpp and imgui_widgets.cpp.
